lib/web_ui/dev/generate_fallback_font_data.dart

```diff
@@ -293,3 +301,23 @@ const List<String> fallbackFonts = <String>[
   'Noto Sans Yi',
   'Noto Sans Zanabazar Square',
 ];
+
+String _packFontRanges(List<String> starts, List<String> ends) {
+  assert(starts.length == ends.length);
+
+  final StringBuffer sb = StringBuffer();
+
+  for (int i = 0; i < starts.length; i++) {
+    final int start = int.parse(starts[i], radix: 16);
+    final int end = int.parse(ends[i], radix: 16);
+
+    sb.write(start.toRadixString(36));
```

Comment:
36 just to get maximum "compression"?

Reply:
Yes. To encode the integer in as few characters as possible.